package.json

```diff
@@ -23,38 +20,41 @@
   "author": "Matt Zabriskie",
```

Comment:
We need to add us as an author and make reference to the original author and fork in the docs/Readme.